src/app/errorReports/withErrorReportsForm.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from 'react';
```

Comment:
I imagine this was taken from comments, but I think for error reports this Higher-Order Component is redundant and could just be removed.

I mean, the data fetching and all that is handled by the parent NewErrorReportForm component anyway (currently it fetches locationData but doesn't actually pass it to the components, but that would just take 1 or 2 lines to fix), and any high-level layout elements needed (i.e. the Modal component, I guess) could be part of NewErrorReportForm too.